Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/core/execution/TestCertificateRetrievalWorker.kt

```diff
@@ -38,6 +40,9 @@ class TestCertificateRetrievalWorker @AssistedInject constructor(
             }
         } catch (e: Exception) {
             Timber.tag(TAG).e(e, "Test result retrieval worker failed.")
+            if (e is CancellationException) {
```

Comment:
I think we should surround the whole `doWork()` function body with try-catch. Otherwise this wouldn't work if the Worker is called in the beginning?

Reply:
Maybe allso call `refreshCleanup()` when the worker fails in Line `27`?

Reply:
I see. Thanks!